packit/patches.py

```diff
@@ -33,27 +33,28 @@ def __init__(
         present_in_specfile: bool = False,
         ignore: bool = False,
         squash_commits: bool = False,
+        patch_id: Optional[int] = None,
         no_prefix: bool = False,
         metadata_defined: bool = None,
     ) -> None:
         """
         Metadata about patch files and relation to the respective commit.
 
-        :param name: name of the patch file
-        :param path: Path of the patch file
-        :param description: will be attached as a comment above path in spec-file
-                            (if present_in_specfile=False)
-        :param commit: git.Commit relevant to this patch file
-        :param present_in_specfile: if the patch is already in the spec-file
-                                    and we don't need to add it there
-        :param ignore: We don't want to process this commit
-                        when we convert source-git commits to patches.
-                        This patch will be skipped.
-        :param squash_commits: squash commits into a single patch file
-                               until next commits with squash_commits=True
-                               (git-am patches do this)
-        :param no_prefix: do not prepend a/ and b/ when generating the patch file
-        :param metadata_defined: are any of the metadata defined in a commit message
+        Args:
+            name: name of the patch file (patch_name is the actual metadata key)
+            path: Path of the patch file
+            description: will be attached as a comment above Patch spec definition
+                         (if present_in_specfile=False)
+            commit: git.Commit relevant to this patch file
+            present_in_specfile: if the patch is already in the spec-file
+                                 and we don't need to add it there
+            ignore: this patch will not be processed.
+            squash_commits: squash commits into a single patch file
+                            until next commits with squash_commits=True
+                            (git-am patches do this)
+            no_prefix: do not prepend a/ and b/ when generating the patch file
+            patch_id: the number from definition PatchNNNN in the spec
+            metadata_defined: are any of the metadata defined in a commit message
```

Comment:
This should be an `Attributes` block on the class level. Would be nice if the order would be the same as the order of args for `__init__()`.

Reply:
my bad with the order change, thanks for pointing that out

will move that up to be a class docstring: took me some time to parse that information from the documentation